src/transformation/builtins/index.ts

```diff
@@ -81,7 +81,8 @@ function tryTransformBuiltinGlobalMethodCall(
 ) {
     const ownerType = context.checker.getTypeAtLocation(calledMethod.expression);
     const ownerSymbol = tryGetStandardLibrarySymbolOfType(context, ownerType);
-    if (!ownerSymbol || ownerSymbol.parent) return;
```

Comment:
Wondering if instead we shouldn't just be removing this `ownerSymbol.parent`, not really sure if it adds anything